docs/ManualDeploymentGuideScalarDLOnAzure-1.md

```diff
@@ -0,0 +1,217 @@
+# Deploy Scalar DL on Azure
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker.
+It can be deployed on various platforms and is recommended to be deployed on managed services for production to achieve high availability, scalability, and maintainability.
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Azure as a starting point for deploying Scalar DL for production.
+
+## What we create
+
+![image](images/azure-diagram.png)
+
+In this guide, we will create the following components.
+
+* An Azure Virtual Network associated with a Resource Group.
+* An AKS cluster with 2 node pools.
+* A managed database service.
+    * A Cosmos DB Account.
+* A Bastion instance with a public IP.
+* Azure container insights.
+
+## Prerequisites
+
+You must have required permissions as specified in [Cloud permissions document](./CloudPermissionsForScalarDLOnAKS.md) to create cloud resources to deploy Scalar DL on Azure.
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards.
+Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production.
+This section shows how to configure a secure network for Scalar DL deployments.
+
+### Requirements
+
+* You must create a virtual network with a subnet for bastion.
+* You must create 2 subnets with the prefix of at least `/22` for AKS, one subnet must be created with the name `k8s_ingress` to create an envoy load balancer.
```

Comment:
I asked a similar question in aws PR, but I think the prefix condition is more appropriate for recommendations.

Also, there are some IP ranges that cannot be used, so it would be better to attach the following link.
https://docs.microsoft.com/en-us/azure/aks/configure-kubenet

Reply:
>I asked a similar question in aws PR

I used prefix used in scalar-terraform.

>  but I think the prefix condition is more appropriate for recommendations.

Moved it to recommendations and also added the link in b2c656c